docs/reference/announcements/870.md

```diff
@@ -90,6 +92,18 @@ Camunda 8.7 introduces a streamlined architecture, consolidating core components
 
 You can download the alpha release of the unified package from the Camunda GitHub repository, either as an executable Java application (Camunda Orchestration Core) or a Docker image.
 
+:::caution breaking change: Deploy diagram modal
```

Comment:
Maybe out of scope for this PR, but wondering if this breaking change should be mentioned anywhere else in our SM documentation? Perhaps the linked documentation here? @conceptualshark may have stronger opinions one way or another here.

Reply:
The original updates were made in https://github.com/camunda/camunda-docs/pull/4688, but I think a note that a configuration _must_ be provided to select a cluster might be in order - I can follow up with that.

Reply:
Thanks @christinaausley and @conceptualshark, I appreciate the great feedback here 🚀